src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -20,7 +20,7 @@
 <div id="workItemList_Error" class="error" *ngIf="error">{{error}}</div>
 
 <div id="workItemList_quickAdd" class="pull-left">
-    <div *ngIf="loggedIn" class="pull-left supress-hover" (click)="addWorkItem()">
+    <div *ngIf="loggedIn" class="pull-left supress-hover">
```

Comment:
We are not using this addWorkItem function any more. I don't know when it was removed but it was throwing an error. In fact the error is currently on production as well. We don't need this event anyway, so removed it.